src/99-appendices/08-coordinate-systems.md

```diff
@@ -112,10 +116,14 @@ below, the origin is at the AC and the orientation of the axes is RAS. Unless
 specified explicitly in the sidecar file in the `XXCoordinateSystemUnits` field,
 the units are assumed to be mm.
 
+### Volume
+
 | Coordinate System                                   | Description                                                                                                                                                                                                                                                                                                                                                      |
-| ---------------------------------------------------   | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------   |
+| --------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| FS305                                               | FreeSurfer variant of the MNI305 space                                                                                                                                                                                                                                                                                                                           |
```

Comment:
This should be implicit to `fsaverage*` for the same reasons of `MNI152NLin6AsymConte69` should not be a space. This example clearly shows how sampling should be detached from the notion of space.

Reply:
This is just a label, but it should not include any special character (such as `*`), so if anything `fsaverage` can be used. I not familiar with this space though - @edickie did you add this one?